assets/tsg/java.tsg.alt

```diff
@@ -160,7 +160,7 @@ attribute refkind = kind                => debug_refkind = kind
   node @this.push_start
 }
 
-(program)@this {
+(program)@_this {
```

Comment:
The latest `tree-sitter-stack-graphs` code is a little more strict. If a capture like this is unused, it must be prefixed with `_`.